### PR TITLE
fix the base fee params for base mainnet

### PR DESCRIPTION
**Pull request by roberto-bayardo**
..also removed the OP_*_DEFAULT constants which were just adding clutter.

### CODE REVIEW

File: crates/primitives/src/constants/mod.rs [hunk 1]
```diff
@@ -60,11 +60,17 @@ pub const EIP1559_DEFAULT_BASE_FEE_MAX_CHANGE_DENOMINATOR: u64 = 8;
 /// Elasticity multiplier as defined in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559)
 pub const EIP1559_DEFAULT_ELASTICITY_MULTIPLIER: u64 = 2;
 
+/// Base fee max change denominator for Optimism.
+pub const OP_GOERLI_EIP1559_DEFAULT_BASE_FEE_MAX_CHANGE_DENOMINATOR: u64 = 50;
+
+/// Elasticity multiplier for Optimism.
+pub const OP_GOERLI_EIP1559_DEFAULT_ELASTICITY_MULTIPLIER: u64 = 10;
+
```
Comment (roberto-bayardo) [inline crates/primitives/src/constants/mod.rs:68]: perhaps we should just get rid of all these OP_*_DEFAULT constants entirely and just allow the optimism() and optimism_goerli() factories to specify them?

Comment (refcell) [reply crates/primitives/src/constants/mod.rs:68]: Yes this seems like the best path forward. @clabby is working on an upstream sync and then we are going to point the pr to upstream reth to start eliciting reviews

Comment (roberto-bayardo) [reply crates/primitives/src/constants/mod.rs:68]: ok let me nuke this before merging then. will have to do it later tonight.

Comment (roberto-bayardo) [reply crates/primitives/src/constants/mod.rs]: done

**Review by refcell — APPROVED** [review]
LGTM

**Review by clabby — APPROVED** [review]

**Pull request merged**